components/Image/Image.tsx

```diff
@@ -2,12 +2,13 @@
 export interface IImageProps {
   src: string
   alt?: string
+  className?: string
 }
 
-const Image = ({ src, alt }: IImageProps) => {
+const Image = ({ src, alt, className }: IImageProps) => {
   return (
     <div className="leading-none">
-      <img className="object-contain" src={src} alt={alt} />
+      <img className={`${className} object-contain`} src={src} alt={alt} />
```

Comment:
move the ${className} on the container div. If the image has "object-contain" it shall be contained inside the div so all width-heights should be done there